flink-cdc-composer/src/main/java/org/apache/flink/cdc/composer/flink/translator/TransformTranslator.java

```diff
@@ -84,10 +92,18 @@ public DataStream<Event> translatePostTransform(
         }
         postTransformFunctionBuilder.addTimezone(timezone);
         postTransformFunctionBuilder.addUdfFunctions(
-                udfFunctions.stream()
-                        .map(udf -> Tuple2.of(udf.getName(), udf.getClasspath()))
-                        .collect(Collectors.toList()));
+                udfFunctions.stream().map(this::udfDefToUDFTuple).collect(Collectors.toList()));
+        postTransformFunctionBuilder.addUdfFunctions(
+                models.stream().map(this::modelToUDFTuple).collect(Collectors.toList()));
         return input.transform(
                 "Transform:Data", new EventTypeInfo(), postTransformFunctionBuilder.build());
     }
+
+    private Tuple3<String, String, Map<String, String>> modelToUDFTuple(ModelDef model) {
+        return Tuple3.of(model.getModelName(), model.getClassName(), model.getParameters());
```

Comment:
Add classpath prefix here to reuse the logic in UDFDescriptior.

Reply:
Done.